File: flutter_basic/lib/backoffice/presentation/boxes/statsData.dart
class StatsData {
  final int onlineUsers;
  final int postsDone;
  final int unhandledReports;
  final int unactivatedAccounts;
  final int unhandledReservations;

  StatsData({
    required this.onlineUsers,
    required this.postsDone,
    required this.unhandledReports,
    required this.unactivatedAccounts,
    required this.unhandledReservations,
  });

  factory StatsData.fromJson(Map<String, dynamic> json) {
    return StatsData(
      onlineUsers: json['onlineUsers'],
      postsDone: json['postsDone'],
      unhandledReports: json['unhandledReports'],
      unactivatedAccounts: json['unactivatedAccounts'],
      unhandledReservations: json['unhandledReservations'],
    );
  }
}

File: src/main/java/pt/unl/fct/di/apdc/firstwebapp/util/entities/clientObjects/StatsData.java
package pt.unl.fct.di.apdc.firstwebapp.util.entities.clientObjects;

public class StatsData {

    private long onlineUsers;
    private long postsDone;
    private long unhandledReports;
    private long unactivatedAccounts;
    private long unhandledReservations;
    private long unhandledAnomalies;
    

    public StatsData() {}

    public StatsData(long onlineUsers, long postsDone, long unhandledReports, long unactivatedAccounts,
            long unhandledReservations, long unhandledAnomalies) {
        this.onlineUsers = onlineUsers;
        this.postsDone = postsDone;
        this.unhandledReports = unhandledReports;
        this.unactivatedAccounts = unactivatedAccounts;
        this.unhandledReservations = unhandledReservations;
        this.unhandledAnomalies = unhandledAnomalies;
    }

    /**
     * @return the onlineUsers
     */
    public long getOnlineUsers() {
        return onlineUsers;
    }

    /**
     * @return the postsDone
     */
    public long getPostsDone() {
        return postsDone;
    }

    /**
     * @return the unhandledReports
     */
    public long getUnhandledReports() {
        return unhandledReports;
    }

    /**
     * @return the unactivatedAccounts
     */
    public long getUnactivatedAccounts() {
        return unactivatedAccounts;
    }

    /**
     * @return the unhandledReservations
     */
    public long getUnhandledReservations() {
        return unhandledReservations;
    }

    /**
     * @return the unhandledAnomalies
     */
    public long getUnhandledAnomalies() {
        return unhandledAnomalies;
    }
    
}
